src/client/src/containers/SelectFrontEndFramework/index.tsx

```diff
@@ -50,7 +50,7 @@ type Props = IDispatchProps & ISelectFrontEndFrameworkProps & InjectedIntlProps;
 const messages = defineMessages({
   selectFrontendFramework: {
     id: "selectFrontendFramework.selectFrontendFramework",
-    defaultMessage: "3. Select a front-end framework."
+    defaultMessage: "Select a front-end framework."
```

Comment:
Notice how you modify "id: "selectFrontendFramework.selectFrontendFramework"," -> Read next comment